### PR TITLE
Sortable course types

### PR DESCRIPTION
**Pull request by janno42**
enables explicitly sorting course types. they were sorted by their german name before.

### CODE REVIEW

**Comment by karyon** [comment]
why would staff users want to manually sort course types? wouldn't sorting by name (of course respecting the current locale) or by usage (number of courses that have that type) be sufficient?

**Comment by karyon** [comment]
ref #1243

**Comment by janno42** [comment]
I want to sort them by importance. Lectures and seminars come first, then some projects, "Other" courses are last. Reason is the filter list on the results page which should be ordered in this way.

**Comment by karyon** [comment]
what about automatically sorting them by the number of courses that have that type? that does not require any user interaction and is less arbitrary than the subjective opinion of a staff member.

**Comment by janno42** [comment]
i would argue it's more or less the same as with degrees. there is kind of a "natural" order of the course types that can be set manually. E.g., "Soft skills colloquium" is used quite often but still less important than every other "regular" course type.

File: evap/staff/forms.py [hunk 1]
```diff
@@ -102,6 +102,11 @@ def clean(self):
 
 
 class CourseTypeForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["order"].widget = forms.HiddenInput()
```
Comment (karyon) [inline evap/staff/forms.py]: i see we use these `HiddenInput`s a lot, but is there a reason why we can't just exclude the field from `fields`?

Comment (janno42) [reply evap/staff/forms.py]: the field must be included in the form to post the new value after reordering and saving the form, doesn't it?

Comment (karyon) [reply evap/staff/forms.py]: oh, i didn't realize that this is the form used on the course type index page.

**Pull request merged**